tsdb/testutil.go

```diff
@@ -12,6 +12,11 @@ import (
 	"github.com/prometheus/prometheus/tsdb/tsdbutil"
 )
 
+const (
+	int_histogram   = "integer histogram"
+	float_histogram = "float_histogram"
```

Comment:
Formatting: It's more common in golang (and in this repo) to use camelCase rather than snake_case

```suggestion
	intHistogram   = "integer histogram"
	floatHistogram = "float_histogram"
```